### PR TITLE
Feature/logging

### LINKED ISSUE #12: Implement logging feature
- [x] Implement logs 
   - [x] Investigate with @josemp10 best way to implement this feature. Auditing api can be found [here](https://docs.dhis2.org/master/en/developer/html/webapi_auditing.html). Problably we will have to save this information using the app store api
   - [x] Implement a div at the top hidden by default. Add button or link to show the content. It should allow scrolling.
   - [x] Retrieve logging from the server.
   - [x] Persist our own logs
   - [x] Define how much information will be displayed and how to display/load so we don't have any performance issue

### PR DESCRIPTION
**Pull request by tokland**
Fixes #12

### CODE REVIEW

**Comment by tokland** [comment]
Some notes:

- For simplicity, I've used a "See more" link instead of a infinite scroll.
- Finally I settled for a circular bucket logging (maxBucket / maxEntriesPerBucket), which is is easy to implement and can limit the total entries saved.

Some screenshots:

logs1
logs2
logs3
logs4

File: index.html [hunk 1]
```diff
@@ -88,6 +88,11 @@
     <script src="core/dhis2Api.js" type="text/javascript"></script>
     <script src="app.js" type="text/javascript"></script>
 
+    <!--Services-->
```
Comment (adrianq) [inline index.html]: @tokland  Is this something you want to keep?

**Review by adrianq — APPROVED** [review]
@tokland looks awesome and works smoothly! Just a minor comment. I am fine to keep the comment just want to be sure there is a reason to do so.

File: index.html (hunk 1 above)
Comment (tokland) [reply index.html]: Nope, out it goes.

**Pull request merged**